src/main/cliapp/src/components/ControlledVocabularyMultiSelector.js

```diff
@@ -2,19 +2,19 @@ import React, {useState} from 'react';
 import { MultiSelect } from "primereact/multiselect"
 
 export function ControlledVocabularyMultiSelectDropdown({ options, editorChange, props, placeholderText }) {
-    const [selectedValues, setSelectedValues] = useState();
+    const [selectedValues, setSelectedValues] = useState(props.rowData.diseaseQualifiers);
```

Comment:
Actually not sure about passing props in.... typically its either all the variables or just props.... but defer to @adamgibs on this one.

Reply:
Technically, I don't think it matters. There's just a prop named props. But it could be confusing since there is a convention of passing in a props object that references all of the props passed to the component and then referencing them as `props.propName`. A possible solution could be to rename the props prop to editorProps or something. But in terms of functionality, it shouldn't matter.